src/components/MainMenu.js

```diff
@@ -54,6 +62,9 @@ export default function MainMenu() {
           </ItemWrap>
         ))}
       </CardLayout>
+      <div>
+        <PackageVersion>{VERSION}</PackageVersion>
+      </div>
```

Comment:
Do we need a div here? Also can I have a `v` in front of the version so it's a bit more obvious it's a version?